README.md

```diff
@@ -608,3 +610,7 @@ understand any regexes.
   providing an implementation.
 * Add warnings for when branches are used in non-popping loops.
 * Fix infinite loop/recursion when rule refers to itself
+
+[^unic]: Valid YAML characters without YAML indicators, whitespace, control,
+ASCII punctuation and symbols, and special purpose and private use planes
+(the full list is in the `is_valid_name_char` function at [common.rs](https://github.com/BenjaminSchaaf/sbnf/blob/master/src/compiler/common.rs))
```

Comment:
There is also an existing reference to `sbnf.sbnf` that contains the `IN` and `OUT` char sets, didn't copy&paste it to the readme since in practice the easiest way to find out is to type a char and see it highlighted in red rather than read the full table somewhere